cg/store/api/find_business_data.py

```diff
@@ -74,10 +74,18 @@ def get_application_by_case(self, case_id: str) -> Application:
             .sample.application_version.application
         )
 
+    def get_application_limitations_by_tag(self, tag: str) -> list[ApplicationLimitations] | None:
```

Comment:
Perhaps out of scope, but this function should be in a read handler

Reply:
Yes, I agree, as some other functions in this class. Maybe we can address it in a future refactoring PR 🤔